src/app/(proper_react)/Dashboard.module.scss

```diff
@@ -0,0 +1,25 @@
+@import "../tokens";
+
+.container {
+  max-width: $content-xl;
+ 
+  h2 {
+    font: $text-body-lg;
+    font-weight: 600;
+  }
+
+  .exposuresArea {
+    margin-top: $layout-sm;
+
+    .exposureList {
+      list-style: none;
+      padding-top: $layout-xs;
+      padding-left: 0;
+      margin: 0;
+  
+      .exposureListItem {
+        margin-bottom: $spacing-md;
```

Comment:
Referring to [composable CSS](https://non-traditional.dev/encapsulated-css-the-key-to-composable-layouts-94f11c177cc1) again, I think a good rule of thumb is that you almost always want to avoid `margin` if you can. So instead of this `margin-bottom`, you'd make the parent a flex vertical container with a `gap` and a `padding-block` if need be. Likewise for the margin on line 12.